### PR TITLE
Feat: Create helm chart test

### PR DESCRIPTION
**Pull request by agg3fe**
## Description

Added Helm test for this service.

### CODE REVIEW

File: charts/bpndiscovery/config/default-realm-import.json [hunk 1]
```diff
@@ -0,0 +1,2006 @@
...
+      "default-client" : [ {
+        "id" : "20368ccb-ace5-4924-bb92-81c2b8cc23a2",
+        "name" : "uma_protection",
+        "composite" : false,
+        "clientRole" : true,
+        "containerId" : "f1c5d623-0c0a-4d33-92b0-adaa9436cb04",
+        "attributes" : { }
+      }, {
+        "id" : "aa76cb61-ed7d-4fca-8f7e-5cb53f70966d",
+        "name" : "add_semantic_model",
```
Comment (tunacicek) [inline charts/bpndiscovery/config/default-realm-import.json:90]: adjust roles for bpn discovery.

File: charts/bpndiscovery/values.yaml [hunk 2]
```diff
@@ -18,6 +18,9 @@
 ###############################################################
 
 enablePostgres: true
+# enables the default keycloak identity provider
+# relies on a postgres instance
+enableKeycloak: true
```
Comment (tunacicek) [inline charts/bpndiscovery/values.yaml:23]: please set this value to false.

File: charts/bpndiscovery/values.yaml [hunk 3]
```diff
@@ -18,6 +18,9 @@
 ###############################################################
 
 enablePostgres: true
+# enables the default keycloak identity provider
+# relies on a postgres instance
+enableKeycloak: false
```
Comment (tunacicek) [inline charts/bpndiscovery/values.yaml:23]: you can remove this parameter

**Comment by agg3fe** [comment]
Not Required

**Pull request closed without merging**